src/docs/Components/Hero/code.mdx

```diff
@@ -0,0 +1,116 @@
+---
+title: 'Code'
+order: 2
+---
+
+## Import
+
+Import *the settings*, *headings*, *the grid* and *the hero* scss files. 
+```scss
+@import 'settings-tools/_all-settings';
+
+@include import-font-families();
+
+@import 'typography/_t.all-base-typography';
+@import 'typography/_t.headings';
+@import 'layouts/_l.flexy';
+@import 'components/_c.hero';
+```
+
+## Basic usage
+A Hero is composed of several zones:
+
+- *A body* with the class: `mc-hero__header`
+- *A header* with the class: `mc-hero__header`
+- *A content* with the class: `mc-hero__content` (optional)
+- *A background image* with the class: `mc-hero__background`
+
+These area are encompassed by a global wrapper class: `mc-hero`
+
+```html
+
+<div class="mc-hero ml-flexy ml-flexy--items-center">
+  <div class="mc-hero__body mc-hero__body--with-content">
+      <div class="mc-hero__header">
+        <p class="mt-heading
+        mt-heading--lightest
+        mt-heading--s
+        mt-heading--underline">Hero title</p>
+    </div>
+    <div class="mc-hero__content mt-body-m">
+      <p> Lorem ipsum dolor sit amet, consectetur adipisicing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua.</p>
+    </div>
+  </div>
+
+  <div class="mc-hero__background">
+    <picture>
+      <img class="mc-hero__img" src="https://m1.lmcdn.fr/media/15/5e611393475ea771bbbad626/2872342965/greenkub-maison-sans-permis5e611393063b330009a4a405.jpg?width=800&crop=16:9,smart" alt="Greenkub house">
+    </picture>
+  </div>
+</div>
+
+```
+
+<Preview path="titleContent" />
+
+### Details of area
+In more detail, it should be noted that each of the fields mentioned above may contain other elements. Thus:
+
+### Main
+In order to have a clean layout you will have to add the following classes to the container (`mc-hero`):
+- `ml-flexy ml-flexy--items-center`
+
+### The background
+The background bloc contains:
+- An image element (`mc-hero__img`) inside a picture element
+- Before the img element you can add all the source for different screen size
+
```

Comment:
Please add an highlight with the complete structure picture + source + image, as an example.